lua/core/menu/devices.lua

```diff
@@ -130,25 +133,41 @@ m.enc = function(n,delta)
   end
 end
 
-local function redraw_keyboard_layout_select()
-  local len = tabutil.count(m.options["keyboard layout"])
+local function redraw_select(do_show_all_if_fit, do_uppercase)
```

Comment:
this fn basically implements your comment https://github.com/monome/norns/pull/1408#issuecomment-909635580